pkg/clients/edgeconnect/config.go

```diff
@@ -0,0 +1,13 @@
+package edgeconnect
+
+import (
+	"github.com/Dynatrace/dynatrace-operator/pkg/util/logger"
+)
+
+var (
+	log = logger.Factory.GetLogger("edgeConnectClient")
+)
+
+var DefaultOauthScopes = []string{"test"}
+
+const DefaultTokenURL = "https://sso-dev.dynatracelabs.com/sso/oauth2/token"
```

Comment:
A dev URL hardcoded seems suspicious, specially in open source code. AFAIK this URL is not reachable outside of our labs.

Reply:
good point, will remove it.

Reply:
fixed.